docs/api/covidcast-signals/nhsn.md

```diff
@@ -0,0 +1,94 @@
+
+State and nation-level values are pulled directly from the source; HHS-level values are aggregated up from the state level.
+
+
+| Signal                          | Description                                                                                                                                                                         |
+|---------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `confirmed_admissions_covid_ew`              | Total number of patients hospitalized with confirmed COVID-19 captured for the Wednesday of the reporting week **Earliest date available:** 2020-08-08                                                                                                     |
+| `confirmed_admissions_covid_ew_prelim`          | Total number of patients hospitalized with confirmed COVID-19 captured for the Wednesday of the reporting week preliminary data reported to NHSN for the previous reporting week (Sunday – Saturday). <br/> **Earliest date available:** 2020-08-08  |
+| `confirmed_admissions_flu_ew`           | Total number of patients hospitalized with confirmed influenza captured for the Wednesday of the reporting week  <br/> **Earliest date available:** 2020-08-08                                                                                       |
+| `confirmed_admissions_flu_ew_prelim`     | Total number of patients hospitalized with confirmed influenza captured for the Wednesday of the reporting week  preliminary data reported to NHSN for the previous reporting week (Sunday – Saturday).<br/> **Earliest date available:** 2020-08-08 |
+
+## Table of Contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+## Estimation
+
+All data is weekly such that each reported value represents a total from Sunday to Saturday of the reference week.
+The `output_signal_name` signal mirrors the `input signal name` field for all geographic resolutions except HHS.
+
+### Geographic weighting
+
+State and nation-level values are pulled directly from the source; HHS-level values are aggregated up from the state level by summing the values of member states.
+
+
+## Missingness
+
+Data for reference dates through April 30, 2024 were reported during a federally-mandated reporting period as specified by the Secretary of the Department of Health and Human Services.
+Data for reference dates May 1, 2024 – October 31, 2024 were voluntarily reported in the absence of a mandate.
+As a result, during this period the total number of hospitalized patients on a subsection of hospitals and may not be fully representative.
+Data for reference dates beginning November 1, 2024 were reported during the current mandated reporting period.
+
+All data and metrics capturing information on respiratory syncytial virus (RSV) were voluntarily reported until November 1, 2024.
+
+
+## Limitations
```

Comment:
suggestion: Any other limitations?

For example, the old HHS said [some types of hospitals and medical facilities were not included](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/hhs.html#limitations). Will that also apply here?